Doc/library/sqlite3.rst

```diff
@@ -745,10 +748,9 @@ Connection Objects
 
    .. attribute:: row_factory
 
-      You can change this attribute to a callable that accepts the cursor and the
-      original row as a tuple and will return the real result row.  This way, you can
-      implement more advanced ways of returning results, such  as returning an object
-      that can also access columns by name.
+      A callable that accepts two arguments,
+      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
+      and return any object that fit the application programmers need.
```

Comment:
I'm rather confused by what

> return any object that fit the application programmers need.

is supposed to mean; aside from the grammar issues, it is unclear to me what I as a user am supposed to do with this.

Reply:
Well. I also find it hard to word this. The row factory can return whatever the programmer wishes. For one application, returning a string may be the right thing. For another application, returning a dataclass, or a complex object, or just a named tuple may be right.

Reply:
Perhaps just: _"return a row object tailored to the application specific need"_

Reply:
`and returns an arbitrary Python object representing a SQLite row`?

Reply:
Hm, "arbitrary" sounds ... arbitrary :) How about "returns a custom object", or "application specific object"?

Reply:
BTW, the "representing an SQLite row" suffix is definitely an improvement. 🐿️

Reply:
> Hm, "arbitrary" sounds ... arbitrary :) How about "returns a custom object", or "application specific object"?

"Custom object representing an SQLite row" works, or you could go with just "object representing an SQLite row"

> 🐿️

Dammit, will I ever remember 😄

Reply:
Great, I'll land this using the former variant. Thank you so much, both of you :)